### PR TITLE
chore(#8956): translate to spanish 600 to 900

### PR DESCRIPTION
**Pull request by latin-panda**
# Description

Translation to Spanish from L600 to L900

#8956

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs
If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8956-translate-to-spanish-600-to-900/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8956-translate-to-spanish-600-to-900/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8956-translate-to-spanish-600-to-900/docker-compose/cht-couchdb-clustered.yml)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: api/resources/translations/messages-es.properties [hunk 1]
```diff
@@ -598,241 +598,306 @@ debug.db_info.name = Nombre
...
+login.token.general.error = Algo salió mal al procesar su solicitud. Inténtalo de nuevo. Si esto vuelve a suceder, comuníquese con el administrador para recibir un nuevo enlace.
+login.token.invalid = Su enlace no es válido. Póngase en contacto con el administrador para recibir un nuevo enlace.
+login.token.loading = Iniciando sesión. Espere por favor.
+login.token.missing = A su enlace le falta información requerida. Póngase en contacto con el administrador para recibir un nuevo enlace.
+login.token.redirect.login = Página de inicio de sesión
+login.token.redirect.login.info = Si conoce su nombre de usuario y contraseña, haga clic en el siguiente enlace para cargar la página de inicio de sesión.
+login.token.timeout = Se agotó el tiempo de espera al intentar iniciar sesión. Inténtelo de nuevo. Si esto vuelve a suceder, comuníquese con el administrador para recibir un nuevo enlace.
+login.unsupported_browser = Para una mejor experiencia con la aplicación, comuníquese con su administrador o supervisor.
+login.unsupported_browser.outdated_cht_android = Hágales saber que actualice el cht-android.
```
Comment (lorerod) [inline api/resources/translations/messages-es.properties:829]:
```suggestion
login.unsupported_browser.outdated_cht_android = Hágales saber que es necesario actualizar la versión del cht-android.
```

File: api/resources/translations/messages-es.properties [hunk 2]
```diff
@@ -598,241 +598,306 @@ debug.db_info.name = Nombre
...
+login.unsupported_browser.outdated_webview_apk = Hágales saber que actualice el Webview APK.
```
Comment (lorerod) [inline api/resources/translations/messages-es.properties:830]:
```suggestion
login.unsupported_browser.outdated_webview_apk = Hágales saber que es necesario actualizar la versión del Webview APK.
```

File: api/resources/translations/messages-es.properties [hunk 3]
```diff
@@ -598,241 +598,306 @@ debug.db_info.name = Nombre
...
+login.unsupported_browser.outdated_browser = Hágales saber que actualice su navegador.
```
Comment (lorerod) [inline api/resources/translations/messages-es.properties:831]:
```suggestion
login.unsupported_browser.outdated_browser = Hágales saber que es necesario actualizar su navegador.
```

File: api/resources/translations/messages-es.properties [hunk 4]
```diff
@@ -598,241 +598,306 @@ debug.db_info.name = Nombre
...
+message.characters.left.multiple.many = {{messages}} SMS, quedan {{characters}} carácteres - \¡demasiados mensajes\!
+message.list.complete = No más mensajes
+messages.c.report_accepted = Gracias {{contact.name}} por registrar a {{patient_name}}. Su ID es {{patient_id}}. Han sido inscritos en el programa de salud infantil.
+messages.c.validation.months_since_birth = El formato de registro para {{patient_id}} es incorrecto, asegúrese de que la edad esté en meses y entre 0 y 59.
+messages.c.validation.patient_name = {{\#patient_name}}El formato de registro es incorrecto. Comience con C seguida de un espacio, edad en semanas seguida de un espacio y el nombre del niño (máximo de 30 caracteres).{{/patient_name}}{{^patient_name}}. Falta el nombre en el formato de registro. Comience con C seguida de un espacio, la edad en semanas seguida de un espacio y el nombre del niño {{/patient_name}}.
+messages.c.validation.weeks_since_birth = El formato de registro para {{patient_id}} es incorrecto, asegúrese de que la edad esté en semanas y entre 0 y 260.
+messages.d.report_accepted = Gracias {{contact.name}} por informar la entrega de {{patient_name}} ({{patient_id}}).
```
Comment (lorerod) [inline api/resources/translations/messages-es.properties:840]: What do you mean with "entrega"? 
If I understand it correctly I think a better term would be "parto" or "alumbramiento".

File: api/resources/translations/messages-es.properties [hunk 5]
```diff
@@ -598,241 +598,306 @@ debug.db_info.name = Nombre
...
 messages.errors.invalid = Mensaje inválido\:
 messages.errors.message.empty = Traducción no encontrada
 messages.errors.patient.missing = Paciente no encontrado
-messages.f.report_accepted =
-messages.f.report_accepted_parent = 
-messages.generic.no_provided_patient_id =
+messages.errors.place.missing = Lugar no encontrado
+messages.errors.unknown.contact = Este contacto ya no existe, no puedes enviarle un mensaje.
+messages.f.report_accepted = Gracias por marcar a {{patient_name}} ({{patient_id}}). Si se trata de una emergencia, envíela a la clínica de inmediato.
+messages.f.report_accepted_parent = {{patient_name}} ({{patient_id}}) fue marcado por {{contact.name}} ({{contact.phone}}) para mayor atención.
```
Comment (lorerod) [inline api/resources/translations/messages-es.properties:849]: What do you mean with "marcado"?
Could it be "puesto en contacto con"?

File: api/resources/translations/messages-es.properties [hunk 6]
```diff
@@ -598,241 +598,306 @@ debug.db_info.name = Nombre
...
+messages.generic.no_provided_patient_id = La compatibilidad con el número de identificación externo está configurada, pero no se encuentra en el documento en la ubicación configurada.
 messages.generic.provided_patient_id_not_unique = La identificación provista ya está en uso.
+messages.generic.provided_phone_not_valid = El teléfono proporcionado no es válido.
 messages.generic.registration_accepted = Gracias {{contact.name}} por inscribir {{patient_name}}. Su identificación es {{patient_id}}.
-messages.generic.registration_not_found = Ninguna persona con Numero de Identificación '{{patient_id}}' encontrado. Verifica el numero de Identificación y envíe el mensaje de nuevo.
+messages.generic.registration_not_found = Ninguna persona con número de identificación '{{patient_id}}' fue encontrado. Verifique el número de identificación y envíe el mensaje de nuevo.
```
Comment (lorerod) [inline api/resources/translations/messages-es.properties:854]:
```suggestion
messages.generic.registration_not_found = Ninguna persona con número de identificación '{{patient_id}}' fue encontrada. Verifique el número de identificación y envíe el mensaje de nuevo.
```

File: api/resources/translations/messages-es.properties [hunk 7]
```diff
@@ -598,241 +598,306 @@ debug.db_info.name = Nombre
...
+messages.off.report_accepted = No se enviarán más notificaciones sobre {{patient_name}} hasta que envíe 'ON {{patient_id}}'.{{\#chw_sms}} {{chw_sms}}{{/chw_sms}}
+messages.on.report_accepted = Las notificaciones para {{patient_name}} ({{patient_id}}) se han reactivado.{{\#chw_sms}} {{chw_sms}}{{/chw_sms}}
+messages.p.report_accepted = Gracias por registrar el embarazo de {{patient_name}} ({{patient_id}}).{{\#expected_date}} Su FEP es {{\#date}}{{expected_date}}{{ /fecha}}{{/fecha_esperada}}
+messages.p.validation.last_menstrual_period = El formato de registro para {{patient_name}} es incorrecto, asegúrese de que FUM sea un número entre 2 y 42.
+messages.relay.chw_sms = {{\#chw_sms}}{{chw_sms}}{{/chw_sms}}
+messages.schedule.anc.checkin = \¿Dónde entregó {{patient_name}}? Responda con 'D {{patient_id}} F' para parto en un centro, 'D {{patient_id}} S' para parto en casa con partera capacitada, 'D {{patient_id}} NS' para parto en casa no especializado.
```
Comment (lorerod) [inline api/resources/translations/messages-es.properties:866]:
```suggestion
messages.schedule.anc.checkin = \¿Dónde ocurrió el parto de {{patient_name}}? Responda con 'D {{patient_id}} F' para parto en un centro, 'D {{patient_id}} S' para parto en casa con partera capacitada, 'D {{patient_id}} NS' para parto en casa no especializado.
```

File: api/resources/translations/messages-es.properties [hunk 8]
```diff
@@ -598,241 +598,306 @@ debug.db_info.name = Nombre
...
+messages.schedule.anc.followup = \¿{{patient_name}} asistió a su visita de atención prenatal\? Cuando lo haga, responda con 'V {{patient_id}}'. Gracias\!
+messages.schedule.anc.reminder = Recuerde a {{patient_name}} ({{patient_id}}) que visite el centro de salud para la visita de atención prenatal esta semana. Cuando lo haga, háganoslo saber con 'V {{patient_id}}'. Gracias\!
+messages.schedule.anc.reminder_due = {{patient_name}} ({{patient_id}}) debe recibir ANC esta semana. Por favor confirme con un mensaje 'V'. Si ha lo recibió, envíe un mensaje 'D'.
```
Comment (lorerod) [inline api/resources/translations/messages-es.properties:869]:
```suggestion
messages.schedule.anc.reminder_due = {{patient_name}} ({{patient_id}}) debe recibir ANC esta semana. Por favor confirme con un mensaje 'V'. Si ya lo recibió, envíe un mensaje 'D'.
```

File: tests/e2e/default/login/login-logout.wdio-spec.js [hunk 9]
```diff
@@ -42,7 +42,7 @@ describe('Login page funcionality tests', () => {
     const esTranslations = {
       user: 'Nombre de usuario',
       pass: 'Contraseña',
-      error: 'Nombre de usuario o contraseña incorrecto. Favor intentar de nuevo.'
+      error: 'Nombre de usuario o contraseña incorrecto. Intentelo de nuevo.'
```
Comment (lorerod) [inline tests/e2e/default/login/login-logout.wdio-spec.js:45]:
```suggestion
      error: 'Nombre de usuario o contraseña incorrecto. Inténtelo de nuevo.'
```

**Review by lorerod — CHANGES_REQUESTED** [review]

File: api/resources/translations/messages-es.properties [hunk 10]
```diff
@@ -598,241 +598,306 @@ debug.db_info.name = Nombre
...
-message.characters.left.multiple.many =
-message.list.complete = 
-messages.c.report_accepted =
-messages.c.validation.months_since_birth = 
-messages.c.validation.patient_name =
-messages.c.validation.weeks_since_birth = 
-messages.d.report_accepted =
-messages.d.validation.days_since_birth = 
-messages.d.validation.delivery_code =
+instance.upgrade.feature_releases = Versiones beta de funcionalidades de prueba
```
[the remark this answers is not in the document]
Comment (latin-panda) [reply api/resources/translations/messages-es.properties:788]: This comes from `feature release`, where we make available experimental features to partners; if it works, it is merged into the main branch and released in CHT. If it doesn't work, we close the branch. 

What do you think is a better name in Spanish?

File: api/resources/translations/messages-es.properties (hunk 5 above)
Comment (latin-panda) [reply api/resources/translations/messages-es.properties:849]: haha yes, thanks!

File: api/resources/translations/messages-es.properties (hunk 4 above)
Comment (latin-panda) [reply api/resources/translations/messages-es.properties:840]: yes, that makes a lot more sense. Thanks!

File: api/resources/translations/messages-es.properties (hunk 5 above)
Comment (latin-panda) [reply api/resources/translations/messages-es.properties:849]: Ah, it says "flagged" like it added some sort of mark or label to the patient, so later, this person can get medical attention. 
So I guess we can use: `{{patient_name}} ({{patient_id}}) fue señalado por {{contact.name}} ({{contact.phone}}) para recibir mayor atención.` ?

**Comment by latin-panda** [comment]
Thanks for the review @lorerod!
I've applied your feedback but I have 2 open questions in your feedback.

File: api/resources/translations/messages-es.properties (hunk 5 above)
Comment (latin-panda) [reply api/resources/translations/messages-es.properties]: I made the change [in this commit ](https://github.com/medic/cht-core/pull/8980/commits/0a1c45c3ccf0ac06b676fba2702527aa6f2052d8) is it okay?

File: api/resources/translations/messages-es.properties (hunk 10 above)
[the remark this answers is not in the document]
Comment (lorerod) [reply api/resources/translations/messages-es.properties:788]: Thank you for the explanation. I get it now. It is ok like it is.

File: api/resources/translations/messages-es.properties [hunk 11]
```diff
@@ -598,241 +598,306 @@ debug.db_info.name = Nombre
...
+messages.c.validation.months_since_birth = El formato de registro para {{patient_id}} es incorrecto, asegúrese de que la edad esté en meses y entre 0 y 59.
+messages.c.validation.patient_name = {{\#patient_name}}El formato de registro es incorrecto. Comience con C seguida de un espacio, edad en semanas seguida de un espacio y el nombre del niño (máximo de 30 caracteres).{{/patient_name}}{{^patient_name}}. Falta el nombre en el formato de registro. Comience con C seguida de un espacio, la edad en semanas seguida de un espacio y el nombre del niño {{/patient_name}}.
+messages.c.validation.weeks_since_birth = El formato de registro para {{patient_id}} es incorrecto, asegúrese de que la edad esté en semanas y entre 0 y 260.
+messages.d.report_accepted = Gracias {{contact.name}} por informar el parto de {{patient_name}} ({{patient_id}}).
+messages.d.validation.days_since_birth = El informe de entrega de {{patient_id}} es incorrecto. Asegúrese de que los días transcurridos desde la entrega sean un número de días entre 0 y 365, o déjelo en blanco.
```
Comment (lorerod) [inline api/resources/translations/messages-es.properties:841]:
```suggestion
messages.d.validation.days_since_birth = El informe del parto del {{patient_id}} es incorrecto. Asegúrese de que los días transcurridos desde el parto sean un número de días entre 0 y 365, o déjelo en blanco.
```

Sorry, I missed this before.

File: api/resources/translations/messages-es.properties [hunk 12]
```diff
@@ -598,241 +598,306 @@ debug.db_info.name = Nombre
...
+messages.d.validation.delivery_code = {{\#delivery_code}}No se entendió el código de entrega. Utilice el código "F" para centro, "S" para parto en casa con partero calificado, "NS" para parto sin SBA.{{/delivery_code}}{{^delivery_code}}Faltaba el código de parto. Utilice el código "F" para centro, "S" para parto en casa con un asistente de parto calificado, "NS" para parto sin un asistente calificado.{{/delivery_code}}
```
Comment (lorerod) [inline api/resources/translations/messages-es.properties:842]:
```suggestion
messages.d.validation.delivery_code = {{\#delivery_code}}No se entendió el código de parto. Utilice el código "F" para centro, "S" para parto en casa con partero calificado, "NS" para parto sin SBA.{{/delivery_code}}{{^delivery_code}}Faltaba el código de parto. Utilice el código "F" para centro, "S" para parto en casa con un asistente de parto calificado, "NS" para parto sin un asistente calificado.{{/delivery_code}}
```

**Review by lorerod — CHANGES_REQUESTED** [review]

**Comment by latin-panda** [comment]
@lorerod Suggestions applied!

**Review by lorerod — APPROVED** [review]
Gracias @latin-panda !

**Pull request merged**